2.0/s2i/bin/run

```diff
@@ -10,6 +10,8 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_vars=$(cgroup-limits.py) ; export $export_vars
```

Comment:
updating sti-base is going to be pretty painful and delay this.  do you really think this is critical?